src/sync/sync_session.cpp

```diff
@@ -423,6 +423,7 @@ void SyncSession::handle_error(SyncError error)
             case ProtocolError::reuse_of_session_ident:
             case ProtocolError::bound_in_other_session:
             case ProtocolError::bad_message_order:
+            case ProtocolError::pong_timeout:
```

Comment:
If I'm reading this correctly, this results in `pong_timeout` being reported to the binding's error handler. Is that really what we want?